test/pkg.jl

```diff
@@ -926,4 +926,24 @@ end
     @test_throws Pkg.Types.PkgError("Package PackageSpec(\n  path = test_packages/Example\n  version = *\n) has neither name nor uuid") ensure_resolved(Pkg.Types.Context(), Pkg.Types.Manifest(), [p])
 end
 
+@testset "Issue #3147"
```

Comment:
```suggestion
@testset "Issue #3147" begin
```

Reply:
Yeah, sorry, draft for now ;)

Reply:
Yep, just meaning to help the test run :)